### DISCUSSION TITLE
AntDesign Blazor

### DISCUSSION THREAD

**Discussion by mehmetuken** [abpframework/abp#6869] [Q&A]
how to usage blazor and blazorise with this library ? https://github.com/ant-design-blazor/ant-design-blazor

**Comment by berkansasmaz** [comment]
I recommend you look [here](https://blazorise.com/docs/usage/ant-design/).

**Comment by ElderJames** [comment]
The components and features of the Ant Design Blazor are much richer than the Ant Design theme of Blazorise. Because Blazorise is constrained by the uniform API, many features of Ant Design have to be sacrificed.

Hopefully the community can help with this. 😉

**Comment by mehmetuken** [comment]
Yes, blazorise ant design future is limited and ant design blazor pro and pro have rich components rich and nice dashboard option.

**Comment by ElderJames** [comment]
You can take a look at this repo first. https://github.com/struggle3/MyAbp

**Reply by mehmetuken** [reply]
First look it's nice, thanks.
